.github/workflows/docker-publish.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 env:
   REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ toLower(github.repository) }}
```

Comment:
https://github.com/orgs/community/discussions/25768

tolower does not exist, we have to use a workaround